lib/l10n/app_es.arb

```diff
@@ -0,0 +1,539 @@
+  "entryActionOpen": "Abrir con…",
+  "entryActionSetAs": "Establecer como…",
+  "entryActionOpenMap": "Mostrar en aplicación de mapa…",
+  "entryActionRotateScreen": "Rotar pantalla",
+  "entryActionAddFavourite": "Agregar a favoritos",
+  "entryActionRemoveFavourite": "Quitar de favoritos",
+
+  "videoActionCaptureFrame": "Capturar fotograma",
+  "videoActionPause": "Pausa",
+  "videoActionPlay": "Reproducir",
+  "videoActionReplay10": "Retroceder 10 segundos",
+  "videoActionSkip10": "Adelantar 10 segundos",
+  "videoActionSelectStreams": "Seleccionar pistas",
+  "videoActionSetSpeed": "Velocidad de reproducción",
+  "videoActionSettings": "Ajustes",
+
+  "entryInfoActionEditDate": "Editar fecha y hora",
+  "entryInfoActionEditRating": "Editar clasificación",
+  "entryInfoActionEditTags": "Editar etiquetas",
+  "entryInfoActionRemoveMetadata": "Eliminar metadatos",
+
+  "filterFavouriteLabel": "Favorito",
+  "filterLocationEmptyLabel": "No localizado",
+  "filterTagEmptyLabel": "Sin etiquetar",
+  "filterRatingUnratedLabel": "Sin clasificar",
+  "filterRatingRejectedLabel": "Rechazado",
+  "filterTypeAnimatedLabel": "Animado",
+  "filterTypeMotionPhotoLabel": "Foto en movimiento",
+  "filterTypePanoramaLabel": "Panorámica",
+  "filterTypeRawLabel": "Sin procesar",
+  "filterTypeSphericalVideoLabel": "Video en 360°",
+  "filterTypeGeotiffLabel": "GeoTIFF",
+  "filterMimeImageLabel": "Imagen",
+  "filterMimeVideoLabel": "Video",
+
+  "coordinateFormatDms": "GMS",
+  "coordinateFormatDecimal": "Grados decimales",
+  "coordinateDms": "{coordinate} {direction}",
+  "coordinateDmsNorth": "N",
+  "coordinateDmsSouth": "S",
+  "coordinateDmsEast": "E",
+  "coordinateDmsWest": "O",
+
+  "unitSystemMetric": "Métrico",
+  "unitSystemImperial": "Imperial",
+
+  "videoLoopModeNever": "Nunca",
+  "videoLoopModeShortOnly": "Sólo videos cortos",
+  "videoLoopModeAlways": "Siempre",
+
+  "mapStyleGoogleNormal": "Mapas de Google",
+  "mapStyleGoogleHybrid": "Mapas de Google (Híbrido)",
+  "mapStyleGoogleTerrain": "Mapas de Google (Superficie)",
+  "mapStyleOsmHot": "OSM Humanitario",
+  "mapStyleStamenToner": "Stamen Monocromático (Toner)",
+  "mapStyleStamenWatercolor": "Stamen Acuarela (Watercolor)",
+
+  "nameConflictStrategyRename": "Renombrar",
+  "nameConflictStrategyReplace": "Reemplazar",
+  "nameConflictStrategySkip": "Saltear",
+
+  "keepScreenOnNever": "Nunca",
+  "keepScreenOnViewerOnly": "Sólo en el visor",
+  "keepScreenOnAlways": "Siempre",
+
+  "accessibilityAnimationsRemove": "Prevenir efectos en pantalla",
+  "accessibilityAnimationsKeep": "Mantener efectos en pantalla",
+
+  "albumTierNew": "Nuevo",
+  "albumTierPinned": "Fijado",
+  "albumTierSpecial": "Común",
+  "albumTierApps": "Aplicaciones",
+  "albumTierRegular": "Otros",
+
+  "storageVolumeDescriptionFallbackPrimary": "Almacenamiento interno",
+  "storageVolumeDescriptionFallbackNonPrimary": "Tarjeta de memoria",
+  "rootDirectoryDescription": "Directorio raíz",
```

Comment:
`rootDirectoryDescription` and `otherDirectoryDescription` are used inside the full sentences below `storageAccessDialogMessage` and `restrictedAccessDialogMessage`, in the placeholder {directory}.

So, for example, `storageAccessDialogMessage` with `rootDirectoryDescription` would give:
"Por favor seleccione Directorio raíz en “SD Card” en la siguiente pantalla para permitir a esta aplicación tener acceso."

Shouldn't `rootDirectoryDescription` be something like `el directorio raíz`, then?

Reply:
Hi.
Both are correct but if that's how it's used then adding _el_ to the string would be fine, as it implies there's only one rootdir.